doc/modules/ROOT/pages/index.adoc

```diff
@@ -130,166 +305,202 @@ kubectl create configmap service-per-pod-hooks -n metacontroller --from-file=hoo
-]
-...
+[source, bash]
+----
+kubectl get service
+NAME             TYPE           CLUSTER-IP       EXTERNAL-IP      PORT(S)          AGE
+hz-hazelcast     LoadBalancer   10.219.246.40    35.230.84.127    5701:30443/TCP   4m2s
+hz-hazelcast-0   LoadBalancer   10.219.255.141   34.145.108.167   5701:30091/TCP   4m7s
+hz-hazelcast-1   LoadBalancer   10.219.241.203   34.82.71.106     5701:30687/TCP   4m5s
+hz-hazelcast-2   LoadBalancer   10.219.247.106   35.247.93.190    5701:32452/TCP   4m4s
 ----
 
-== Configure Hazelcast Client outside Kubernetes
+We'll use the `hz-hazelcast` service for the discovery.
+
+[source, bash]
+----
+kubectl get service hz-hazelcast
+NAME           TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)          AGE
+hz-hazelcast   LoadBalancer   10.219.246.40   35.230.84.127   5701:30443/TCP   5m29s
+----
 
-When we have a working Hazelcast cluster deployed on Kubernetes, we can connect to it with an external Hazelcast Client. You need first to fetch the credentials of the created Service Account and then use them to configure the client.
+[NOTE]
+====
+If you are using Minikube, you need to execute `minikube tunnel` now in order to get LoadBalancer External IPs assigned.
+====
 
-=== Check Kubernetes Master IP
+The field `EXERNAL-IP` is the address of your Hazelcast cluster.
 
-To check the IP address of the Kubernetes Master, use the following command.
+=== Connecting with Hazelcast Client
 
-[source, shell script]
+Configure the Hazelcast client to connect to the cluster external address.
+
+[tabs]
+====
+
+Java::
++
+--
+[source, java]
 ----
-$ kubectl cluster-info
-Kubernetes master is running at https://35.233.44.52
+ClientConfig config = new ClientConfig();
+config.getNetworkConfig().addAddress("<EXTERNAL-IP>");
 ----
+--
 
-=== Check Access Token and CA Certificate
+NodeJS::
++
+--
+[source, javascript]
+----
+const { Client } = require('hazelcast-client');
 
-First, you need to find the name of the secret for the created Service Account.
+const clientConfig = {
+    network: {
+        clusterMembers: [
+            '<EXERNAL-IP>'
+        ]
+    }
+};
+const client = await Client.newHazelcastClient(clientConfig);
+----
+--
 
-[source, shell script]
+Go::
++
+--
+[source, go]
 ----
-$ kubectl get secret
-NAME                                    TYPE                                  DATA   AGE
-default-token-7xd6t                     kubernetes.io/service-account-token   3      102m
-hazelcast-service-account-token-m8p82   kubernetes.io/service-account-token   3      35m
+import (
+	"log"
+
+	"github.com/hazelcast/hazelcast-go-client"
+)
+
+func main() {
+    config := hazelcast.NewConfig()
+	config.ClusterConfig.SetAddress("<EXTERNAL-IP>:5701")
+	client, err := hazelcast.StartNewClientWithConfig(config)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
 ----
+--
+====
 
-Then, to fetch Access Token, use the following command.
+Finally, start the client application using the following command.
```

Comment:
nit: the same consideration on the expected client behavior and output as in the unisocket section.